modules/vpc/variables.tf

```diff
@@ -38,4 +38,24 @@ variable "single_nat_gateway" {
 variable "enable_nat_gateway" {
   description = "Enalbe or disable the creation of the NAT Gateway"
   type        = bool
-}
\ No newline at end of file
+}
+
+variable "vpc_endpoint_service_acceptance_required" {
```

Comment:
This name is confusing, can be changed to `accept_vpc_endpoint_connection_requests`